lib/sdf-server/src/service/change_set/apply_change_set.rs

```diff
@@ -85,6 +86,9 @@ pub async fn apply_change_set(
         }),
     );
 
+    ctx.write_audit_log(AuditLogKind::ApplyChangeset {}, "-".to_string())
```

Comment:
Ah I think we'll need the backend API to make this optional after all... I'll fix that in a follow-up.